source/docs/advanced/shipping/mondial-relay.md

```diff
@@ -9,7 +9,7 @@ This page contains information about the different ways you can ship orders with
 
 No matter the backend solution, the MondialRelay Front-Commerce modules requires a Map component to display pickup points. Thus, before proceeding, please make sure that you've chosen a Map implementation from [Display a map](/docs/advanced/features/display-a-map.html).
 
-## Magento2 based application
+## Integrate with Magento2
```

Comment:
check links to #Magento2-based-application there are two in the migration guide that needs to be updated

Reply:
Thanks, yea I forgot to do that 💡